### PR TITLE
Day3

### PR DESCRIPTION
**Pull request by ashishk144**
@praveengarimella

### CODE REVIEW

File: Day3/templates/Admin.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Admin Page</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+</head>
```
Comment (praveengarimella) [inline Day3/templates/Admin.html:6]: generalize the common HTML code into a generic template and import it.

File: Day3/application.py [hunk 2]
```diff
@@ -9,15 +9,23 @@
 Session = sessionmaker(bind = engine)
 session = Session()
  
+@app.route("/admin")
+def admin():
+   usrs =   session.query(User).all()
+   # for i in usrs:
+      # print(i)
```
Comment (praveengarimella) [inline Day3/application.py:16]: remove commented code

File: Day3/application.py [hunk 3]
```diff
@@ -27,16 +35,18 @@ def register():
...
+      data = session.query(User).filter_by(user_id=user).all()
+      if(len(data) > 0):
+      	return "User already exists"
       if(pwd == pwdcheck):
          try:
             usr = User(user_id = user, pwd = pwd, timestamp = datetime.now())
             session.add(usr)
             session.commit()
-            return redirect(url_for('success',name = user, pwd = pwd))
+            return redirect(url_for('success',name = user))
```
Comment (praveengarimella) [inline Day3/application.py:46]: why not directly render the success page here instead of redirecting?

File: Day3/application.py [hunk 4]
```diff
@@ -9,15 +9,23 @@
 Session = sessionmaker(bind = engine)
 session = Session()
  
+@app.route("/admin")
+def admin():
+   usrs =   session.query(User).all()
```
Comment (praveengarimella) [inline Day3/application.py:14]: does  this show the list in the desc order of the user creation time

File: Day3/application.py [hunk 5]
```diff
@@ -15,9 +15,9 @@ def default():
 
 @app.route('/success/<name>?<pwd>')
 def success(name, pwd):
-   print("name ", name)
-   print("pwd", pwd)
-   return 'welcome ' + name + " "+ pwd
+   # print("name ", name)
+   # print("pwd", pwd)
```
Comment (praveengarimella) [inline Day3/application.py:26]: remove commented code

File: Day3/users.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+#         pwd = db.column(db.String, nullable = False)
+#         timestamp = db.column(db.String, primary_key = True)
+
+# def main():
+#   db.create_all()
+#   app.run(debug = True)
+
+
+
+# engine = create_engine('postgres://dzfsgqgmtgypmj:a6b4fbac38a9bb0a87ffbe7b4af6855d067998f6dd8cfb85621c87d309ca9d4d@ec2-18-213-176-229.compute-1.amazonaws.com:5432/d8muda130eogs8', echo = True)
```
Comment (praveengarimella) [inline Day3/users.py:43]: Don't include passwords and private keys in the git repo.

**Review by praveengarimella — CHANGES_REQUESTED** [review]
Pls see my comments, resolve them and merge

File: Day3/application.py (hunk 3 above)
Comment (ashishk144) [reply Day3/application.py:46]: That was to re-route Sir, is it okay to render the template are /register URL itself?

File: Day3/application.py (hunk 4 above)
Comment (ashishk144) [reply Day3/application.py:14]: updated the code sir, will commit after all the changes.

File: Day3/users.py (hunk 6 above)
Comment (ashishk144) [reply Day3/users.py:43]: Removed all the superfluous code sir

File: Day3/application.py (hunk 5 above)
Comment (ashishk144) [reply Day3/application.py:26]: removed the code sir, will commit after all the changes.

File: Day3/application.py (hunk 2 above)
Comment (ashishk144) [reply Day3/application.py:16]: removed the code sir, will commit after all the changes.

File: Day3/templates/Admin.html (hunk 1 above)
Comment (ashishk144) [reply Day3/templates/Admin.html:6]: There isn't any common layout which I'm following yet Sir. As the title for each page is different and I'm using different CSS files as well

**Pull request merged**